frontend/mgramseva/lib/utils/role_actions.dart

```diff
@@ -123,10 +120,6 @@ class RoleActionsFiltering {
         return ['EXPENSE_PROCESSING', 'SUPERUSER'];
       case Routes.EXPENSE_UPDATE:
         return ['EXPENSE_PROCESSING', 'SUPERUSER'];
-

```

Comment:
can you attach the ticket link for this ?

Reply:
Sure